stream/src/test/java/org/apache/apex/malhar/stream/api/impl/ApexStreamImplTest.java

```diff
@@ -80,7 +81,11 @@ public void testAddOperator()
       // Assert the stream is from first operator to second operator
       Assert.assertEquals("first", stream.getSource().getOperatorMeta().getName());
       Assert.assertTrue(1 == stream.getSinks().size());
-      Assert.assertEquals("second", stream.getSinks().get(0).getOperatorWrapper().getName());
+      Collection<InputPortMeta> portMetaList = stream.getSinks();
+      Assert.assertEquals(1, portMetaList.size());
```

Comment:
Remove assert on line 83, it checks for the same condition. Avoid 'List' in the name, it is a Collection.